programs/mango-v3-reimbursement/src/state/table.rs

```diff
@@ -0,0 +1,20 @@
+use std::mem::size_of;
+
+use anchor_lang::prelude::*;
+use static_assertions::const_assert_eq;
+
+#[account(zero_copy)]
+pub struct Table {
+    pub rows: [Row; 1], // TODO: change to 32000
+}
```

Comment:
I'd not make the table fixed-size. It shouldn't be hard to just deal with UncheckedAccount, check the size and then index into the bytes, bytemucking each individual row. That would allow cheap test setups on mainnet without needing to adjust the code.